src/snap-confine.apparmor.in

```diff
@@ -54,6 +54,13 @@
 
     # changing profile
     @{PROC}/[0-9]*/attr/exec w,
+    # Reading current profile
+    @{PROC}/[0-9]*/attr/current r,
+
+    # Probably to find where apparmor is mounted
```

Comment:
You can drop the "probably" here. It is used to find where securityfs is mounted.